openshift/template.yaml

```diff
@@ -48,6 +48,11 @@ objects:
             value: "${CLASS_TYPE}"
           - name: FLASK_LOGGING_LEVEL
             value: ${FLASK_LOGGING_LEVEL}
+          - name: SENTRY_DSN
```

Comment:
Do we have this secret already in stage and prod?

Reply:
If secrects are in cluster namespace then Yes, This key is used for `bayesian-api` also.
If Secrects are for individual workspace, then No